src/api/app/assets/stylesheets/webui/requests.scss

```diff
@@ -37,3 +37,12 @@
     @extend .text-break;
   }
 }
+
+.card#request {
```

Comment:
Instead of having an ID only to set some CSS rules in another file, we could rely on the Bootstrap spacing utility classes to achieve the same. See my other comments below.